src/Compilers/CSharp/Test/Semantic/Semantics/SemanticErrorTests.cs

```diff
@@ -24833,5 +24833,36 @@ public class C {
                 //             return (1, 2);
                 Diagnostic(ErrorCode.ERR_CantConvAnonMethReturns, "(1, 2)").WithArguments("lambda expression").WithLocation(12, 20));
         }
+
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/68110")]
+        public void DefaultSyntaxValueReentrancy_01()
+        {
+            var source =
+                """
+                #nullable enable
+
+                [A(3, X = 6)]
+                public interface A
+                {
+                    public int X { get; set; }
+
+                    public void M(int x, A a = new A()) { }
```

Comment:
>public void M(int x, A a = new A()) { } [](http://example.com/codeflow?start=20&length=39)

Is this used? If not, consider removing.

Reply:
Yes, I'm explicitly pulling on the default parameter value below.